Vagrantfile

```diff
@@ -20,16 +20,22 @@ DISTRO=$(lsb_release -is | tr '[:upper:]' '[:lower:]')
 CODENAME=$(lsb_release -cs)
 echo "deb http://repos.mesosphere.io/${DISTRO} ${CODENAME} main" | sudo tee /etc/apt/sources.list.d/mesosphere.list
 
-apt-get -q -y update
+apt-get -o Acquire::http::Timeout=1 -o Acquire::ftp::Timeout=1 -y update
 echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | sudo /usr/bin/debconf-set-selections
-apt-get -q -y install oracle-java8-installer
-apt-get -q -y install oracle-java8-set-default
-apt-get -q -y install libcurl3
-apt-get -q -y install zookeeperd
-apt-get -q -y install aria2
```

Comment:
The reason I put `-q` in is to suppress complaints about stuff already existing when I reprovision (as you do when you iteratively develop).
Perhaps instead of just removing `-q` we should instead check for the existence of the installed package, and if it isn't installed we proceed to do the package installation without `-q`.